src/test/java/de/retest/web/PageFrameIT.java

```diff
@@ -31,6 +31,8 @@ void page_frame_html_should_be_checked( final WebDriver driver ) throws Exceptio
 		final Path simplePagePath = Paths.get( "src/test/resources/pages/page-frame.html" );
 		driver.get( simplePagePath.toUri().toURL().toString() );
 
+		Thread.sleep( 1000 );
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "Thread.sleep()". ([squid:S2925](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS2925))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=84&issues=AWahP8EYrChX7_AlIBKD&open=AWahP8EYrChX7_AlIBKD)

[ISSUE_KEY]: AWahP8EYrChX7_AlIBKD

Reply:
@roesslerj I would say we ignore these smells in SonarCloud as I have no idea how to do it better. The Selenium API for (implicit & explicit) waits doesn't seem to fit our needs here since we are waiting for the entire page, not just particular elements. What do you think?

Maybe in the future we can extend our API with something like `check( Object, String, long)`, where the last param defines the amount of time to wait for the check itself, such that our users don't get those warnings.